sdk/keyvault/test-resources.json

```diff
@@ -77,7 +77,7 @@
     "attestationSite": "[concat(parameters('baseName'), 'site')]",
     "attestationImageUri": "[concat('DOCKER|azsdkengsys.azurecr.io/', parameters('attestationImage'))]",
     "kvName": "[parameters('baseName')]",
-    "hsmApiVersion": "2020-04-01-preview",
+    "hsmApiVersion": "2022-07-01",
```

Comment:
I am curious, is that necessary?

Reply:
Ah, I didn't mean to make this change in this PR. I wasn't able to deploy a Managed HSM for testing without this change.